org.openhab.binding.zigbee/src/main/resources/OH-INF/thing/xiaomi/xiaomi_lumisensor-switch.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zigbee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- These devices do not correctly report their clusters, so they need a static thing definition for the binding to inject the correct clusters into the ZigBeeNode -->
+
+	<thing-type id="xiaomi_lumisensor-switch" listed="false">
+		<label>Xiaomi LUMI Mijia Button</label>
+		<description>Xiaomi LUMI Mijia Button (round)</description>
```

Comment:
Ideally the description should be different to the label :). Can we say what this is (eg Single round button) - I'm no idea, but it would be good to describe the device. We also shouldn't have the manufacturer name in the description when it's in the label.

Reply:
I will adjust it.